postgraphiql/src/components/PostGraphiQL.js

```diff
@@ -375,6 +400,7 @@ class PostGraphiQL extends React.PureComponent {
           ...(this.state.explain && POSTGRAPHILE_CONFIG.allowExplain
             ? { 'X-PostGraphile-Explain': 'on' }
             : null),
+          'X-PostGraphile-Explain-Options': buildExplainOptionsHeader(this.state.explainOptions)
```

Comment:
I wonder if we should just replace the `{ 'X-PostGraphile-Explain': 'on' }` header with
```
{ 'X-PostGraphile-Explain': `on; ${buildExplainOptionsHeader(this.state.explainOptions)}` }
```

?